packages/lightning-core/transactions/TransactionsListItem.js

```diff
@@ -60,10 +57,10 @@ export const TransactionsListItem = ({ type, hash, memo, amount, status }) => {
       </div>
       <div style={{ ...styles.text, ...styles.column }}>
         <div style={{ ...styles.big, ...styles.overflow }}>
-          { hash }
+          <Text bold>TxID: </Text>{ hash }
         </div>
         { memo ? (
```

Comment:
This syntax below is probably cleaner if you don't have anything to return if the conditional is false :)

`{memo && <div style={{ ...styles.small, ...styles.overflow }}>Note: { memo }</div>}`